src/main/scala/edu/ie3/simona/agent/grid/GridAgent.scala

```diff
@@ -64,6 +64,8 @@ object GridAgent extends DBFSAlgorithm {
         listener,
         resolution,
         simStartTime,
+        TimeUtil.withDefaults
+          .toZonedDateTime(simonaConfig.simona.time.endDateTime),
```

Comment:
`simonaConfig.simona` is called three times in this method. `.time` is called twice. Maybe we could savie them to variables.